downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -109,35 +109,13 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### New features
```

Comment:
Sorry, one last change. This should probably come under improvements rather than new features?

Reply:
Fixed with [21ba8dd8c3a061eb1718a6a5894de146dace15a0](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/21ba8dd8c3a061eb1718a6a5894de146dace15a0)